src/applications/vaos/tests/appointment-list/redux/selectors.unit.spec.js

```diff
@@ -261,56 +261,6 @@ describe('VAOS selectors', () => {
       );
     });
 
-    it.skip('should return next day’s schedule if current time is after window start', () => {
```

Comment:
We've been skiping this test forever, I figured we can just go ahead and delete it.